lib/charms/postgresql_k8s/v1/postgresql.py

```diff
@@ -164,6 +164,7 @@ def _configure_pgaudit(self, enable: bool) -> None:
             connection = self._connect_to_database()
             connection.autocommit = True
             with connection.cursor() as cursor:
+                cursor.execute("RESET ROLE;")
```

Comment:
PgBouncer needs to reset its user from `pgbouncer_owner` to `relation-X` (which is a superuser) to call `ALTER SYSTEM...`.